documentation/api/query/v4/reports.markdown

```diff
@@ -310,3 +310,79 @@ this route.
 This query endpoint supports paged results via the common PuppetDB paging
 URL parameters.  For more information, please see the documentation
 on [paging][paging].
+
+## Differences in Puppet Enterprise
+
+### Response format
+
+The response format in PE contains an additional field, `resources`, which
+contains all the unchanged resources for the Puppet run corresponding to the
```

Comment:
Wait, isn't it all resource states? Not just unchanged resources? I mean, resources that change will also be represented here won't they?